docs/tutorial/recent-documents.md

```diff
@@ -39,11 +39,36 @@ will be started with the path of the file added as a command line argument.
 
 ## macOS Notes
 
-When a file is requested from the recent documents menu, the `open-file` event
-of `app` module will be emitted for it.
+__Adding the Recent Documents list to the application menu:__
```

Comment:
This can be an `h3`:

`### Adding the Recent Documents list to an Application Menu:`